src/resolver/resolver.zig

```diff
@@ -49,7 +49,11 @@ const DotEnv = @import("../env_loader.zig");
 
 pub fn isPackagePath(path: string) bool {
```

Comment:
This should be split into two functions

```js
isPackagePath
isPackagePathNotAbsolute
```

and the version used in resolver.zig should be `isPackagePathNotAbsolute`

To avoid the duplicate work of checking if its an absolute path twice